packages/components/src/mobile/bottom-sheet/stepper-cell/README.md

```diff
@@ -0,0 +1,93 @@
+StepperCell
+===================
+
+`StepperCell` shows a stepper control to change a value wrapped in a `Cell` component.
+
+## Usage
+
+Usage example
+
+```jsx
+import StepperCell from './bottom-sheet/stepper-cell';
+
+function Stepper( { onChangeValue, value } ) {
+	return (
+        <StepperControl
+            icon="screenoptions"
+            label="Columns"
+            maxValue={ 8 }
+            minValue={ 1 }
+            onChangeValue={ onChangeValue }
+            value={ value }
+        />
+	);
+}
+```
+
+## Props
+
+### icon
+
+Dashicon key name of the icon.
+
+- Type: `String`
+- Required: No
+- Platform: Mobile
+
+### label
+
+Label of the component.
+
+- Type: `String`
+- Required: No
+- Platform: Mobile
+
+### maxValue 
+
+Maximum value of the stepper.
+
+- Type: `Number`
+- Required: Yes
+- Platform: Mobile
+
+### minValue 
+
+Minimum value of the stepper.
+
+- Type: `Number`
+- Required: Yes
+- Platform: Mobile
+
+### step 
+
+Step increment value.
+
+- Type: `Number`
+- Required: No
+- Platform: Mobile
+
+### separatorType 
+
+Separator type for the `Cell` style.
+
+- Type: `String`
+- Required: No
+- Platform: Mobile
```

Comment:
This kind of docs looks great! Thank you for including it.

But this is something I'm not too familiar with, for example:
`separatorType` is a prop of the base `Cell` component. As far as I understand, in this file we should document the props that are particular for the `stepper` component, and `inherit` properties (even from docs) from the parent component.

Does it makes sense or maybe I'm lost? :)

Reply:
I think we should add the docs for the StepperControl instead of StepperCell, because StepperControl is the public API. And for inheriting the docs, it can only be possible if parent view is also part of the public `@wordpress/components` API. Base `Cell` is not public and i think it won't be public. If we had long term plans for `separatorType` I'd suggest adding a public parent view but we want to [get rid of that](https://github.com/wordpress-mobile/gutenberg-mobile/issues/1421) one as soon as possible so not sure if it's worth it.

Reply:
> I think we should add the docs for the StepperControl instead of StepperCell

That's what I thought at the beginning, but I was surprised that there wasn't docs already for `StepperCell`.

Also `icon` and `label` are also inherited props. Maybe we want to use BaseControl as an abstraction of our base Cell component (and also extract the TextInput logic from it).

Reply:
Thanks for the feedback! It makes sense to move the docs to `StepperControl` and remove the props that are not specific for the component itself. I'll start with that but it's also good to know how we could improve it in the future.